.gitignore

```diff
@@ -0,0 +1,329 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Wow! This default gitignore that people have put together for VS has really got out of hand, 329 lines.

To keep it simple, let's go with the first 11 lines of Castle Core's gitignore.

Reply:
Done, but I included `packages/` or it will try check-in the NuGets.